.github/workflows/main.yml

```diff
@@ -190,12 +211,20 @@ jobs:
       - name: Build and test Veracruz on IceCap
         run: bash /work/veracruz/icecap/ci/run.sh
 
+     # Upload Cargo.lock files, using the action from github
+      - name: Upload Cargo.lock files
```

Comment:
Just a note that "workflow artifacts" and "release artifacts" are handled a bit differently.

> By default, GitHub stores build logs and [workflow] artifacts for 90 days

If we want these artifacts to persist they will need to eventually be attached to a release

Reply:
Ah got it, a good use of artifacts then